.github/workflows/test-cdk-cd.yml

```diff
@@ -1,24 +1,56 @@
-name: Test tiles generation CDK CD and ECS task run
+name: Test tiles generation CDK and ECS task run
 
-on: 
-  pull_request_target:
-      branches:
-        - main
-      paths:
-        - 'tiles-generation/**'
+on:
+  workflow_run:
+    workflows: ["Receive PR Test Approve"]
+    types:
+      - completed
 
 jobs:
-  approve:
+  comment:
```

Comment:
what does this action imply?

Reply:
I moved the approve step to `pr-approve-test.yml` workflow, here the `comment` in `test-cdk-cd.yml` will help to post the GH actions build url on PR comment.

Reply:
Here is a example https://github.com/junqiu-lei/maps/pull/14